src/app/App.jsx

```diff
@@ -1,13 +1,15 @@
 import React, { Fragment } from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import { Home } from './home';
+import { Signup } from './signup';
 import Footer from './footer/Footer';
```

Comment:
Add an index.js to the footer directory so that we can import from './footer' directly.